src/environment/ssDaylight.lua

```diff
@@ -11,7 +11,7 @@ ssDaylight = {}
 g_seasons.daylight = ssDaylight
 
 function ssDaylight:load(savegame, key)
-    self.latitude = Utils.clamp(ssXMLUtil.getXMLFloat(savegame, key .. ".environment.latitude", 51.9), 0, 72)
+    self.latitude = Utils.clamp(ssXMLUtil.getXMLFloat(savegame, key .. ".environment.latitude", 47.4), 0, 72)
```

Comment:
oh that´s my latitude
Was a test , whether the times adjust
and yes they do ,so everybody can put in his own latitude
i play a while and the weather data is good in game,that fits